CIP-1694/README.md

```diff
@@ -0,0 +1,624 @@
+
+This distribution will determine how much stake is backed by each `yes` vote from a DRep. The exact stake snapshot used for block production by the SPOs will also be used for the DReps for voting.
+
+#### Definitions surrounding Voting Stake
+
+We define some notions around voting stake:
+
+* Lovelace contained in a transaction output is considered **active for voting** if:
+  * It contains a registered stake credential.
+  * The stake credential has delegated its voting rights to a registered DRep.
+* Relative to some given percentage `P`, we say that there is **sufficient active voting stake** if
+  the ratio of the *total active voting stake* to the *total stake in circulation* is at least `P`.
+* Relative to a percentage `P`, a DRep (SPO) **vote threshold has been met** if the sum of the relative stake controlled by the DReps (SPOs) voting 'yes' to a governance action minus the sum of the relative stake controlled by the DReps (SPOs) voting 'no' is at least `P`.
+
+> **Note**
+> There are several alternative definitions for "the total stake in circulation":
+>  1. The sum of the UTxO and the rewards accounts.
+>  2. The sum of the UTxO, the rewards accounts, the fee pot, and the deposit pot.
+>  3. The total ADA supply (i.e. 45 billion ADA) minus the reserves[^3].
+>  We leave the choice open for discussion.
+
+## Rationale
+
+### Role of the Constitutional Committee
+
+At first, the Constitutional Committee may sound like a special committee that's granted extra power over DReps. However, because DReps can replace a committee at any time and DReps votes are also required to ratify every governance action, the Constitutional Committee has no more (and may, in fact, have less) power than the DReps. Given this, what role does the committee play, and why isn't it superfluous? The answer is that the committee solves the bootstrapping problem of this new governance framework. Indeed, as soon as we pull the trigger and enable such a framework to become active on-chain, without a Constitutional Committee, there would rapidly need to be sufficient DReps, so that the system did not rely solely on SPO votes. We cannot yet predict how active the community will be in registering as DReps, nor how reactive other Ada holders will be regarding delegation of votes.
+
+Thus, the Constitutional Committee comes into play to make sure that the system can transition from its current state into fully decentralized governance in due course. Furthermore, in the long run, the committee can play a mentoring and advisory role in the governance decisions by being a set of elected representatives who are put under the spotlight for their judgment and guidance in governance decisions.
+Above all, the committee is required at all times to adhere to the Constitution and to ratify proposals in accordance with the provisions of the Constitution.
```

Comment:
Hi, Excuse my even daring to enter this conversation at this point. Personally my perspextive is very Macro. And determining how to develope a constitutional framework for Cardano eccosystem, (big job), would be essential in my mind and my interest. Am I in the right place? So that rules (code) serve the values of the constitution and can be evaluated in that light, should our constitution be developed 1st?

Reply:
No need to excuse yourself, and yes you are in the right place. I cannot further elaborate on your point though!